docs/LED_Effect.md

```diff
@@ -326,6 +326,14 @@ the "head" of the comet and the remaining colors are blended into the "tail"
     Palette:          Color of "head" and gradient of "tail"
 Identical settings as Comet, but with multiple lights chasing each other.
 
+#### Cylon
+    Effect Rate:  1   How fast the cylon sweeps
+    Cutoff:       0   Not used but must be provided
+    Palette:          Color of sweeping LED
+A light moves between each end of the strip. Multiple colors can be provided;
+the color will change after each sweep. If an odd number of colors are
+provided, the effect will be repeated.
```

Comment:
Please clarify a bit better what is meant by "repeated". I think it will still change the color after each sweep, but for odd numbers you have to go twice to be at the starting color and position. I wonder if it is even necessary to mention it here. It is behaving as expected for me.

Reply:
Sure, I agree it's not really necessary to clarify. Removed.